client/src/pages/EditProfile.js

```diff
@@ -57,11 +58,40 @@ function EditProfile() {
         }
     }
 
-    const updateUIState = (data) => {
+    const updateUIState = async (data) => {
+        console.log('data pipeline: ', data.pipeline)
         setFirstName(data.firstName)
         setLastName(data.lastName)
         setSchool(data.school)
         setAnonymous(data.anonymous)
+        for (const pipeline of data.pipeline) {
+            const response = await fetch(
+                `${HOST}/api/company/get/${pipeline.company}`,
+                {
+                    method: 'GET',
+                    headers: {
+                        'Content-Type': 'application/json', // Specify the content type as JSON
+                    },
+                }
+            )
```

Comment:
I tested this call by making it locally: "localhost:4000/api/company/get/doordash + google" and it failed due to the change you made to the get call in the backend, where it requires an ID instead of a name. Is this a logic thing, or am I misunderstanding the changes?

Reply:
Yeah the get requests are made via company ID now due to schema changes but I probably should also add a get request via name option.

Reply:
I could do that for you king

Reply:
It would just entail copying 3 lines from the pipeline controller file